packages/openlmis/src/Utils.js

```diff
@@ -0,0 +1,98 @@
+import { composeNextState } from '@openfn/language-common';
+import {
+  request as commonRequest,
+  logResponse,
+  makeBasicAuthHeader,
+} from '@openfn/language-common/util';
+
+export const authorize = state => {
+  const {
+    baseUrl,
+    access_token,
+    username,
+    password,
+    clientId = 'user-client',
+    clientSecrete = 'changeme',
+  } = state.configuration;
+
+  const headers = makeBasicAuthHeader(clientId, clientSecrete);
+
+  if (access_token) return state;
+
+  if (username && password) {
+    if (clientId === 'user-client')
+      console.warn('Using default client id:', clientId);
+    if (clientSecrete === 'changeme')
+      console.warn('Using default client secrete:', clientSecrete);
+
+    const options = {
+      query: { grant_type: 'password', username, password },
```

Comment:
Password goes in the query string? For real?

Reply:
Yep, i was very confused as well 😕